layouts/partials/head.html

```diff
@@ -3,40 +3,31 @@
   <meta name="robots" content="all,follow">
   <meta name="googlebot" content="index,follow,snippet,archive">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{{ .Title }}</title>
-  <meta name="author" content="{{ .Site.Author.name }}" />
-
-  {{ if .Keywords }}
-  <meta name="keywords" content="{{ delimit .Keywords ", " }}">
-  {{ else if .Site.Params.DefaultKeywords }}
-  <meta name="keywords" content="{{ delimit .Site.Params.DefaultKeywords ", " }}">	
-  {{ end }}
-
-  {{ if .Description }}
-  <meta name="description" content="{{ .Description }}">
-  {{ else if .Site.Params.DefaultDescription }}
-  <meta name="description" content="{{ .Site.Params.DefaultDescription }}">	
-  {{ end }}
+  <title>{{ .Title | markdownify | plainify }}</title>
```

Comment:
these meta changes are not related with the PR

Reply:
ok, you're right, they're not related to OG/Twitter tags. But honestly: It's just a bunch of senseless additional work to split that up if you're willing to integrate those additional changes anyway. And they're _really not that complicated_ to understand.

So instead: Please just tell me which changes you don't approve and I can remove them.

Reply:
> these meta changes are not related with the PR

Meanwhile it's actually an integral part of this PR since the same variable `$title_plain` is used across different meta tags.